moped-editor/src/components/ApolloErrorHandler.js

```diff
@@ -24,7 +25,7 @@ const ApolloErrorHandler = props => {
 
   // Error Variables
   const error = props?.error ?? null;
-  const errorString = JSON.stringify(error);
+  const errorString = JSON.stringify(error, Object.getOwnPropertyNames(error));
```

Comment:
https://stackoverflow.com/questions/18391212/is-it-not-possible-to-stringify-an-error-using-json-stringify

this never worked, the errorString was always going to be `{ }`

Reply:
good test of the error boundary!